cddl/intrep-actx.cddl

```diff
@@ -0,0 +1,4 @@
+ir-appraisal-context = {
```

Comment:
I am not clear why this data structure is needed ..?

We should just have a Staging Area, kept somewhere out side of ACS and then highlight one concept that is known as ACS.

Active Context of Appraisal is the ACS Data Structure, which tracks the Appraisal.

Appraisal starts with reception of Evidence and ACS starts getting populated.

Then ACS keeps growing and at the end, ACS is converted into Attestation Results.

Reply:
I have it as a data structure primarily to have a single object of process. We then say that Appraisal = phases567(leastfixedpoint(Phases234)(Phase1(inputs)))

if the staging area and ACS don't change after running through phases 2, 3, and 4, then there are no more conditional rules that can apply, and you're ready to move to the later phases.